public/app/features/alerting/unified/components/rule-editor/AlertTypeSection.tsx

```diff
@@ -4,19 +4,17 @@ import { Cascader, FieldSet, Field, Input, InputControl, stylesFactory, Select,
 import { config } from 'app/core/config';
```

Comment:
don't change this file, I'm working on it on my branch :)

Reply:
My rebase got a little hairy so there may be some things that got checked in by accident. Mostly just needed to add the new `DataSourceSelect` component